environment.go

```diff
@@ -110,6 +112,15 @@ func (env *environment) getRuntimeGOOS() string {
 	return runtime.GOOS
 }
 
+func (env *environment) getPlatform() string {
+	if runtime.GOOS == "windows" {
+		return "windows"
+	}
+	p, _, _, _ := host.PlatformInformation()
```

Comment:
Is there a reason we do this rather than `lsb_release -a` and parse the content?
I feel like this is overkill to include a full lib for limited functionality

Reply:
Never min, figured it out. This will have to be done.

Reply:
using that library opens up some other possibilities for other segments - like number of running jobs or memory in use.